Config/DefaultEditor.ini

```diff
@@ -5,4 +5,7 @@ SimpleMapName=/Game/FirstPersonBP/Maps/FirstPersonExampleMap
 bAllowClassAndBlueprintPinMatching=true
 bReplaceBlueprintWithClass= true
 bDontLoadBlueprintOutsideEditor= true
-bBlueprintIsNotBlueprintType= true
\ No newline at end of file
+bBlueprintIsNotBlueprintType= true
```

Comment:
This blueprint is in fact not a blueprint

Reply:
fuck you then and see you tomorrow